kii_thing_if.c

```diff
@@ -684,6 +686,30 @@ static kii_bool_t prv_onboard_with_vendor_thing_id(
             return KII_FALSE;
         }
     }
+    if (firmware_version != NULL) {
+        if (kii_api_call_append_body(kii, ",\"firmwareVersion\":",
+                        CONST_STRLEN(",\"firmwareVersion\":")) != 0) {
```

Comment:
I think here should be `",\"firmwareVersion\":\""`, because firmwareVersion is string